tgui/packages/tgui/interfaces/DnaConsole.js

```diff
@@ -1005,7 +1003,22 @@ const GenomeSequencer = props => {
         {buttons[i + 1]}
       </Box>
     );
+
+    if (((i % 8) === 0) && (i !== 0)) {
+      pairs.push(
+        <Box
+          inline
```

Comment:
missing key, also remove extra parentheses from if

Reply:
Necessary changes have hopefully been made.

Reply:
by extra i mean all of them, they are not doing anything.